<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblFind">
           <property name="text">
            <string>Tex to find:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtFind">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblReplace">
           <property name="text">
            <string>Replace with:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtReplace"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioUp">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioDown">
           <property name="text">
            <string>Down</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkCaseSensitive">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkWholeWords">
         <property name="text">
          <string>Whole words only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkRegEx">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSearchResults">
         <property name="text">
          <string>Results</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnFind">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFindNext">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFindPrevoius">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReplace">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReplaceAll">
         <property name="text">
          <string>Replace all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
